doc/nrf/glossary.rst

```diff
@@ -0,0 +1,447 @@
+   It routes most traffic on the Internet.
+
+Internet Protocol version 6 (IPv6)
+   The latest version of the Internet Protocol (IP).
+   It is the communications protocol that provides an identification and location system for computers on networks and routes traffic across the Internet.
+
+IPv4 address
+   A numerical label that is used to identify a network interface of a computer or other network node participating in an IPv4 computer network.
+
+IPv6 address
+   An alphanumerical label that is used to identify a network interface of a computer or other network node participating in an IPv6 computer network.
+
+Kconfig file
+   A configuration file for a module, written in the Kconfig language syntax.
+   It defines build-time configuration options, also called symbols, namely application-specific values for one or more kernel configuration options.
+   It also defines how they are grouped into menus and sub-menus, and dependencies between them that determine what configurations are valid.
+
+Kconfig fragment
+   A configuration file used for building an application image with or without software support from specific Kconfig options.
+   Examples include things like whether to add networking support or which drivers are needed by the application.
+
+Kconfig language
+   A `kernel configuration language <Kconfig language>`_ used in Kconfig files and fragments.
+   It was initially created for the Linux kernel.
+
+Kconfig option
+   A kernel configuration option used in a Kconfig file or fragment.
+
+Kconfig project configuration
+   A Kconfig fragment, usually called :file:`prj.conf`, used to define default Kconfig options for an application.
+   These are foundational options for the application that will always be applied to its built image.
+   However, they can be overridden by applying an additional Kconfig fragment at build time.
+
+Link Layer (LL)
+   “A control protocol for the link and physical layers that is carried over logical links in addition to user data.” Bluetooth Core Specification, Version 5.1, Vol 1, Part A, Section 1.2
+
+Low-Noise Amplifier (LNA)
+   In a radio receiving system, an electronic amplifier that amplifies a very low-power signal without significantly degrading its signal-to-noise ratio.
+
+LTE-M
+   An open standard that is most suitable for medium-throughput applications requiring low power, low latency, and/or mobility, like asset tracking, wearables, medical, Point of Sale (POS), and home security applications.
+   Also known as Cat-M1.
+
+Lightweight Machine to Machine (LwM2M)
+   An application layer protocol.
+   It defines the service architecture for IoT devices and the protocol for device management.
+
+Man-in-the-Middle (MITM)
+   A man-in-the-middle attack is a form of eavesdropping where communication between two devices is monitored and modified by an unauthorized party who relays information between the two devices giving the illusion that they are directly connected.
+
+Manifest file
```

Comment:
Should this be prefixed with "West" or "(West)" , a manifest file is used in other situations as well, one being DFU. Same goes for `Manifest repository`

Reply:
fixed